docs/src/man/comparisons.md

```diff
@@ -0,0 +1,97 @@
+# Comparison with other survey analysis tools
+
+There are multiple alternatives that offer survey analysis tools, most notably
+[SAS/STAT](https://support.sas.com/rnd/app/stat/procedures/SurveyAnalysis.html)
+and [R](https://CRAN.R-project.org/package=survey).
```

Comment:
```suggestion
[SAS](https://support.sas.com/rnd/app/stat/procedures/SurveyAnalysis.html), [STATA](https://www.stata.com/),
and [R](https://CRAN.R-project.org/package=survey).
```

Reply:
I added the link in the suggestion above, but I changed it to the link containing the survey analysis tool.